### ISSUE TITLE
TVDB v4 scraper pulls foreign poster art or not at all 

### ISSUE THREAD

**Issue by bcline1113** [thetvdb/metadata.tvshows.thetvdb.com.v4.python#24]
Running Kodi 19 on the nvidia shield pro 2019, and the TVDB v4 scraper pulls foreign poster art or not at all . Tried reinstalling with no success. Please advise

**Comment by KarellenX** [comment]
Not enough information. Provide a debuglog and the link to the show.

**Comment by bcline1113** [comment]
What do you suggest if I find the debug log doesn’t show any errors, the episodes & shows match fine, yet if it is just poster art that is being incorrectly downloaded, or not being downloaded at all? The attached pic is from Home Improvement. I saw that this occurred on a few others too 
![69248988-E178-4421-9812-5992AA1AB834](https://user-images.githubusercontent.com/62736397/199481331-ab47dfa5-be11-4bef-add1-5370f741f54e.jpeg)

**Comment by KarellenX** [comment]
>What do you suggest 

I suggest you provide what was asked. I asked you for a debuglog, not if there are any errors in it. I asked for the link to the tv show, not a screenshot.

**Comment by bcline1113** [comment]
Just an update. Kodi is still using the TVDB v4 scraper to scan the shows in one of my other shares in. I have gone through the list of shows, and for the most part, the correct poster art has been pulled in. However, there are some that haven’t. Since, as I type this, it’s still scanning, I will check when it’s done to get you a full list of tv shows that have foreign language poster art. Please be patient. I’ll try to figure out how to get he log file off the shield too. Ok?

**Comment by KarellenX** [comment]
Here, this might help with the log... https://kodi.wiki/view/Log_file/Easy
Section 3.3 shows the address to the log on your Shield.
Just make sure debug mode is enabled and you capture the scraping of at least one of the shows.

**Comment by bcline1113** [comment]
I found this link where you commented on. The post date was November 2020. 2 years later, and users still have same problem . The issue is on TVDB end, not mine 

https://github.com/xbmc/metadata.tvdb.com.python/issues/18

**Comment by bcline1113** [comment]
So I rescraped one share of TV Shows on my server using The TVDB (new), and it scraped just about all the info and poster art correctly. I will test the other share soon. Since it seems that the season poster issue is on TVDB's end, would a workaround be just to find poster art and change it manually? Out of curiousity, since I found that link dated November 2020, even though I know the devs/authors of the TVDB APIs for Kodi have other jobs, why hasnt the issue by identified and corrected on TVDBs end being that it first seems to have occurred 2 years ago?

**Comment by bcline1113** [comment]
OK. So I rescraped that TV Share on my server and TVDB v4 scraped in all shows good, aside from one mis match, which I fixed. 2 others had no season posters, but I think I figured that out too. A workaround is visiting theposterdb.com and downloading the season/specials poster art PNG files, then converting to a jpg file. I do need to get a second drive to move some other TV Shows around, but can you agree to keep this open until I get that drive and we know everything is good?

**Comment by antheaezzell** [comment]
@bcline1113 can I close this? It sounds like yes?

**Comment by bcline1113** [comment]
Yes you can. However, I did notice something. When I went to this page, https://thetvdb.com/series/the-world-at-war/artwork/posters, and the link to the page itself, I noticed while accessing the site on my desktop, that a German poster art was the first one listed. So I figured maybe the TVDB v4 API with Kodi is pulling down the first poster it finds, irrelevant of what the language is set for in the settings. Could this be possible? As I said, I did a workaround by going to theposterdb.com and getting the posters from there.

**Comment by dgasaway** [comment]
I am having this issue.  I get German-language season posters with no option to choose English-language posters, even though I have the scraper set to English.  Here is a log where I re-scraped a show.
[https://paste.kodi.tv/uxonizetof](https://paste.kodi.tv/uxonizetof)
If it helps, I also made a log where I brought up the list of available posters for one season.
[https://paste.kodi.tv/usacocihel](https://paste.kodi.tv/usacocihel)
Note, these logs are from a system running Kodi v19, only because it was easier to get logs from this system.  I have the same issue on another system running the latest Kodi.  I'll make the effort to get logs from the other system if it helps.

**Comment by GreenImp** [comment]
I'm experiencing this as well.

**Comment by antheaezzell** [comment]
I've ticketed this for investigation @GreenImp 

You won't be able to access the link, but it is for our reference. https://whipmedia.atlassian.net/browse/TVD-3770

**Comment by GreenImp** [comment]
Cheers, much appreciated!

**Comment by GreenImp** [comment]
@antheaezzell just to let you know, I've noticed that as well as bringing in images from the wrong language, or only a single image (Even though there are many others), it also seems to be sometimes bringing in the wrong type of image.

I've got several TV shows, where the season image is using a banner image instead.

**Comment by vnnegt** [comment]
> I am having this issue. I get German-language season posters with no option to choose English-language posters, even though I have the scraper set to English. Here is a log where I re-scraped a show. https://paste.kodi.tv/uxonizetof If it helps, I also made a log where I brought up the list of available posters for one season. https://paste.kodi.tv/usacocihel Note, these logs are from a system running Kodi v19, only because it was easier to get logs from this system. I have the same issue on another system running the latest Kodi. I'll make the effort to get logs from the other system if it helps.

this is happening for me as well

scraper settings set to English, but i routinely get German language artwork with no option to retrieve any other artwork remotely

**Comment by fermaton** [comment]
I've experienced this behavior too, where some season posters get the German version. Seems random

**Comment by antheaezzell** [comment]
I've raised the priority on this issue @fermaton. We haven't been able to get to it thus far, but hope to soon.
